infrastructure/ess/template.yaml

```diff
@@ -125,6 +125,126 @@ Resources:
     Properties:
       UserName: !Ref S3BucketUser
 
+  ESSAuroraAcp:
+    Type: "AWS::RDS::DBCluster"
+    Properties:
+      MasterUsername: !Ref ESSDBUsername
+      MasterUserPassword: !Ref ESSDBPassword
+      DBClusterIdentifier: aurora-postgresql-cluster
```

Comment:
I think these need to all be different. Looks like we can leave the field out though and AWS will generate one for us. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-dbclusteridentifier

Reply:
Actually this ID is what shows up in the logs, so maybe just add the DB name to the end of each so we can distinguish them?

Reply:
Why should pw and un need to be different?
I mean for good security and hygine they _should_ but we're dealing with a test app here, and the outputs are a bit of a pain. Plus we'd need to copy em back into the `kustomizer`, so I'm trying to cut a corner for speed...

but wondering how AWS would even know there's the same username, wouldn't this be like different companies running RDS and happening to have the same details?